lib/index.js

```diff
@@ -523,6 +536,26 @@ export default {
 		this.toolbar.update({ buildInProgress: true });
 	},
 
+	takeScreeshot() {
+		const filename = atom.config.get('appcelerator-titanium.general.screenshotPath') + "screenshot_" + new Date().getTime() + '.png';
+		const exec = require('child_process').exec;
+
+		if (os.platform() === 'darwin') {
+			// ios only
+			// xcrun simctl io booted screenshot
```

Comment:
Will you implement this part as well / do you have access to a Xcode env?

Reply:
yes, I'll give it a try next week!